wrappers/ortools.rb

```diff
@@ -418,9 +418,12 @@ def solve(vrp, job, thread_proc = nil, &block)
       routes = vrp.routes.collect{ |route|
         next if route.vehicle.nil? || route.mission_ids.empty?
 
+        service_ids = corresponding_mission_ids(services.collect(&:id), route.mission_ids)
+        next if service_ids.empty?
+
         OrtoolsVrp::Route.new(
           vehicle_id: route.vehicle.id,
-          service_ids: corresponding_mission_ids(services.collect(&:id), route.mission_ids)
```

Comment:
What is expected behaviour when there is a shipment in route.mission_ids ?
For now, users can not provide _pickup_ neither _delivery_ extension is _mission_ids_ because it would raise `Each mission_ids should refer to an existant service or shipment` DiscordantProblemError (from vrp create function)

Reply:
If a user uses `shipment` type, then they will have to give the shipment id in the route because that is the only id that they have. The ids with delivery/pickup suffixes are dangerous to expose since they are internal and might change any time.

I guess, if they are using the `shipment` type, and they want to give an initial route then they can supply the shipment_id twice and since it is shipment type, then we are sure that the first is pickup and the second is delivery.